static/disruption_test_flow.svg

```diff
@@ -0,0 +1,16 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1401.75 709.5" width="1401.75" height="709.5">
```

Comment:
This is an annoying aspect of `excalidraw`, if you remove the `height` and `width` it'll be easier to manage and respect the `div`s container

Reply:
got it, thanks!